docs/deployment/README.md

```diff
@@ -8,6 +8,6 @@ ShowWhy is split between a front-end web-based client and a containerized Python
 * A worker node, deployed as a docker container with extra worker configuration.
 
 ## Deployment Options
-We currently deploy the application using Azure Kubernetes Services (AKS). The instructions for deploying the application using AKS are [here](./AKS_DEPLOY.md). This method uses authentication.
+We currently deploy the application using Azure Kubernetes Service (AKS). You can deploy an AKS container directly to your Azure subscription with one click, [here are the instructions](./azure-scripts/README.md) for deployting the application container directly into your Azure subscription.
```

Comment:
deployting => deploying. 

I'd rephrase to:

Our team deploys ShowWhy using Azure AKS. See the [Azure Deployment documentation](./azure-scripts/README.md) for details.